cmd/omctl/cmd/update.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+
+	"github.com/checkr/openmock"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var updateCmd = &cobra.Command{
+	Use:   "update",
+	Short: "update remote directory",
+	Long:  "update remote directory with new / updated components",
+	Args:  cobra.MaximumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// load local templates
+		localOpenMock := openmock.OpenMock{}
+		localOpenMock.ParseEnv()
+		localOpenMock.TemplatesDir = localDirectory
+
+		localOpenMock.SetupLogrus()
+		localOpenMock.SetupRepo()
+
+		err := localOpenMock.Load()
+		if err != nil {
+			logrus.Fatalf("%s: %s", "failed to load yaml templates for mocks", err)
```

Comment:
Should this return err and let Cobra handle failing out?

Reply:
let's return instead of Fatal (they have the same effect in `RunE`), it makes the function easier to test.

Reply:
will fix this